### PR TITLE
Adding  a few more alt exhchange path/pairs to ccxt

### PR DESCRIPTION
**Pull request by topherbuckley**

### CODE REVIEW

File: src/dali/plugin/pair_converter/ccxt.py [hunk 1]
```diff
@@ -74,14 +77,17 @@
...
 # Currently supported exchanges
 _BINANCE: str = "Binance.com"
+_BINANCEUS: str = "Binance US"
 _COINBASE_PRO: str = "Coinbase Pro"
 _GATE: str = "Gate"
 _HUOBI: str = "Huobi"
 _KRAKEN: str = "Kraken"
 _UPBIT: str = "Upbit"
+_COINBASE: str = "Coinbase"
+_OKEX: str = "Okex"
```
Comment (macanudo527) [inline src/dali/plugin/pair_converter/ccxt.py:87]: Could you put these in alphabetical order?

File: src/dali/plugin/pair_converter/ccxt.py [hunk 2]
```diff
@@ -74,14 +77,17 @@
...
 _FIAT_EXCHANGE: str = "Exchangerate.host"
 _DEFAULT_EXCHANGE: str = _KRAKEN
-_EXCHANGE_DICT: Dict[str, Any] = {_BINANCE: binance, _COINBASE_PRO: coinbasepro, _GATE: gateio, _HUOBI: huobi, _KRAKEN: kraken, _UPBIT: upbit}
+_EXCHANGE_DICT: Dict[str, Any] = {_BINANCE: binance, _BINANCEUS: binanceus, _COINBASE_PRO: coinbasepro, _GATE: gateio, _HUOBI: huobi, _KRAKEN: kraken, _UPBIT: upbit, _COINBASE: coinbase, _OKEX: okex}
```
Comment (macanudo527) [inline src/dali/plugin/pair_converter/ccxt.py:90]: Same here. Could you put these in alphabetical order?

**Review by macanudo527 — CHANGES_REQUESTED** [review]
Just some nits.

**Comment by topherbuckley** [comment]
> Just some nits.
> 
> And could you address the pylint error?

I fixed the line too long error via black, but there was also this error at the bottom of the report:
>src/dali/plugin/pair_converter/ccxt.py:90:0: C0301: Line too long (199/160) (line-too-long)

Not sure if this will show up again or not, but the closest thing I found was [this closed issue](https://github.com/pylint-dev/pylint/issues/4662) in the pylint dev repo, which says this issue was resolved. 

There was a mypy error that came after clearing the rest of the issues. I force pushed a rebase to take out the duplicate coinbase class def in the stub. That test just needs to be rerun (I ran it locally and it passed). 

The alphabetical ordering (in addition to the one in the imports that you didn't mention in your review) was also pushed. Let me know if you need anything else on this.

**Review by macanudo527 — APPROVED** [review]
Thanks for making the changes!

LGTM!

**Pull request merged**